### ISSUE TITLE
java_paths is null

### ISSUE THREAD

**Issue by jiandongjia** [twitter/heron#924]
https://github.com/twitter/heron/blob/master/scripts/get_all_heron_paths.sh#L70

heron/scripts/get_all_heron_paths.sh
![xxx](https://cloud.githubusercontent.com/assets/19700922/16077479/b8b54ace-332b-11e6-8e48-440218ee5bd2.jpg)

Line 70 should modified as follows ? 
The second JAVA_PATHS to lowercase
```
java_paths=$(echo "${java_paths}" | fgrep -v "/objc_tools/")
```

**Comment by billonahill** [comment]
Seems reasonable. Would you please provide a patch?

**Comment by caofangkun** [comment]
@jiandongjia  This issue has been fixed and merged , Please close it .Thank you .

**Comment by maosongfu** [comment]
Fixed: #940
